src/bin/health/helm_ivp_thread.cpp

```diff
@@ -73,9 +73,12 @@ jaiabot::apps::HelmIVPStatusThread::HelmIVPStatusThread(
 
 void jaiabot::apps::HelmIVPStatusThread::issue_status_summary()
 {
-    glog.is_debug2() && glog << group(thread_name()) << "Status: " << status_.DebugString()
+    glog.is_debug2() && glog << group(thread_name()) << "Status: " << previous_status_.DebugString()
```

Comment:
I think you can safely publish status_ here but I'm not sure how much different it'll make (just delays error reporting by loop() interval).